app/src/main/java/com/example/trempel/ui/categories/CategoryViewModel.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.trempel.ui.categories
+
+import androidx.databinding.ObservableBoolean
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.trempel.CategoryRepository
+import com.example.trempel.network.CategoryDomainModel
+import io.reactivex.disposables.Disposable
+import javax.inject.Inject
+
+internal class CategoryViewModel @Inject constructor(
+    private val categoryRepository: CategoryRepository
+    ) :
+    ViewModel() {
```

Comment:
strange formatting, check https://www.droidcon.com/2021/10/01/flexible-recyclerview-adapter-with-mvvm-and-data-binding/

Reply:
done